tests_v2/test_ssh_remote_image_drop_upload.py

```
@@ -0,0 +1,247 @@
+    proc = subprocess.run(
+        [cli, "--socket", SOCKET_PATH, "--json", "--id-format", "both", *args],
+        capture_output=True,
+        text=True,
+        env=env,
+        check=False,
+    )
+    if proc.returncode != 0:
+        raise cmuxError(f"cmux {' '.join(args)} failed: {(proc.stdout + proc.stderr).strip()}")
+    try:
+        return json.loads(proc.stdout or "{}")
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)